x-pack/plugin/vectors/src/main/java/org/elasticsearch/xpack/vectors/mapper/DenseVectorFieldMapper.java

```diff
@@ -227,7 +227,8 @@ public String toString() {
 
     public static final TypeParser PARSER = new TypeParser(
         (n, c) -> new Builder(n, c.indexVersionCreated()),
-        notInMultiFields(CONTENT_TYPE)
+        notInMultiFields(CONTENT_TYPE),
+        Version.CURRENT.minimumCompatibilityVersion()
```

Comment:
Given that there are a few of these multi-parameter calls, maybe we should add a super constructor that takes the Builder and verifier functions and passes `Version.CURRENT.minimumCompatibilityVersion()` as a default value?

Reply:
ok, fixed in [07234b1](https://github.com/elastic/elasticsearch/pull/85059/commits/07234b14ffdd56aec368f3569f761d8753dfea4c)